lib/dal/src/prop.rs

```diff
@@ -751,31 +748,28 @@ impl Prop {
             .ok_or(PropError::MapOrArrayMissingElementProp(prop_id))
     }
 
-    pub async fn find_child_prop_index_by_name(
+    pub async fn find_child_prop_id_by_name(
         ctx: &DalContext,
-        node_index: NodeIndex,
+        parent_node_id: Ulid,
```

Comment:
I had some comments earlier that this should be PropId, but reading further into the code I see it can be a SchemaVariantId as well. Oy!